osi_object.proto

```diff
@@ -264,6 +273,66 @@ message StationaryObject
             //
             COLOR_WHITE = 10;
         }
+
+        //
+        // \brief Attributes of type emitting structure. The horizontal_angle and the vertical_angle in
+        // emitted_spatial_intensity are symmetrical across the normal, which is defined by the mounting position
+        // of the emitting structure.
+        //
+        message EmittingStructureAttribute
+        {
+            // This message determines the range of the emitted wavelength and its
+            // desired number of samples.
+            //
+            repeated WavelengthData wavelength_data = 1;
+
+            // Original intensities of emitted electromagnetic wave referred to a discrete
+            // wavelength defined in WavelengthData. This maximum value is weighted based
+            // on the emitted spatial intensity. The size of max_intensitiy_per_wavelength has to be
+            // equal to the samples in wavelength_data.
+            //
+            // Unit: W/m^2
+            //
+            // \note max_intensity_per_wavelength.size() = WavelengthData.samples_number.size()
+            //
+            repeated double max_intensity_per_wavelength = 2;
+
+            // Spatial distribution of the emitted electromagnetic wave's intensity based on the intensity_per_wavelength as
+            // maximum value. For every sample in wavelength_data an emitted_spatial_intensity has to be defined.
+            //
+            // \note emitted_spatial_intensity.size() = WavelengthData.samples_number.size()
+            //
+            repeated SpatialIntensity emitted_spatial_intensity = 3;
+
+            //
+            // \brief Definition of a spatial intensity distribution with a horizontal and a vertical angle
+            // and the corresponding intensity related to a maximum intensity.
+            //
+            message SpatialIntensity
```

Comment:
@LukasElster 
Please move the `message SpatialIntensity` to osi_common.proto.
It is rather generic and will be used in the SETLevel project e.g. in the LidarSensorViewConfiguration to describe the lidar signal emission there (will be used e.g. for ray tracer config).

Moving it now will avoid the effort to move it later ;-)

Reply:
Actually, @ClemensLinnhoff and I discussed this today and we strongly believe that a signal strength in dBm would fit best here (intensity is rather a power distribution in [watts per square metre (W/m2)](https://en.wikipedia.org/wiki/Intensity_(physics))).
It would mean to get rid of the maximum intensity as well, which is on the pro-side, as well, I guess.

Therefore, we recommend the following for osi_common:
```
//
// \brief Definition of a spatial signal strength distribution
// for an emitting / transmitting / receiving entity
// with a horizontal and a vertical angle
// and the corresponding signal strength in dBm (decibels per milliwatt).
//
message SpatialSignalStrength
{
    // Horizontal angle (azimuth) of emission / transmission / reception
    // in the entity's coordinate system.
    //
    // Unit: rad
    //
    optional double horizontal_angle = 1;

    // Vertical angle (elevation) of emission / transmission / reception
    // in the entity's coordinate system.
    //
    // Unit: rad
    //
    optional double vertical_angle = 2;

    // Emitted / transmitted /received signal strength
    // of the emitting / transmitting / receiving entity
    // at the previously defined horizontal and
    // vertical angle for one specific wavelength.
    // The value for the signal strength
    // is given in dBm (decibels per milliwatt).
    //
    // Unit: dBm
    //
    optional double signal_strength = 3;
}
```

Reply:
@PhRosenberger I integrated your recommendations